drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -102,21 +102,30 @@ public Capabilities getCapabilities(WebDriverConfiguration configuration, boolea
 
         new ChromeDriverBinaryHandler(capabilities).checkAndSetBinary(performValidations);
 
+        // verify binary capabilities
+        if (Validate.nonEmpty(binary)) {
+            if (performValidations) {
+                Validate.isExecutable(binary, "Chrome binary must point to an executable file, " + binary);
+            }
+        }
+
+        setChromeOptions(configuration, capabilities);
+
+        return capabilities;
+    }
+
+    public static void setChromeOptions(WebDriverConfiguration configuration, DesiredCapabilities capabilities) {
```

Comment:
It doesn't have to be static - you can call 
`new ChromeDriverFactory().setChromeOptions(configuration, desiredCapabilities);`